apps/laboratory/src/components/Ethers/Ethers5GetCallsStatusTest.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, Stack, Text, Input } from '@chakra-ui/react'
```

Comment:
All `Ethers5....tsx` files are copy pasted from the Ethers examples which is replaced with `ethers@5.x`

Reply:
Can't we reuse the `Ethers...` components for this?

Reply:
Would be super nice actually but the `ethers@5.x` have different type definitions for providers than `ethers@6.x`

Reply:
What is our strategy here? Maintaining various versions will make our codebase more complex and extensive. When we introduced ethers6, we decided to not support any of the new features on ethers5.

Reply:
We still have motivation to continue supporting ethers5. See Slack thread: https://walletconnect.slack.com/archives/C03RVH94K5K/p1723739386027169

But better way to unify ethers and ethers5 examples. This is possible as we discussed with Glitch before..